### ISSUE TITLE
New here. Some questions about pricing parts and placing Parts. 

### ISSUE THREAD

**Issue by karlack26** [KSP-RO/RP-0#1046]
Hello all i have started to learn how to do RP-0 part configs with the part browser. I made my first commit and got it merged yesterday. But ultimately I want to add more sstu parts that have RO configs but are not placed into RP-1 yet. RO configs for any parts are beyond me at this point. 
My questions is about parts that are technically possible and could have been made but were not. 
So for example the 3 man sstu landing pod. 
currently its a RP-0 no cost part Its been placed in the 92 rover mode.
So would moving this part say to 1980s tech be a no no.  
Some thing like that  could have been built then but never was.  
If players wanted to do some alt history route and go interplanetary or building moon bases have a part like this in the 80s would be great. 
I just know form reading what devs have posted over the years in various forms and threads that you like to stick to real hardware as much as possible. 
if you guys have some general rules of thumb for above mentioned parts I would like to hear them or if you think i should just stick to parts that have real world equivalents.

**Comment by pap1723** [comment]
Welcome @karlack26 !
Since you are officially a "modder" now, welcome into the wonderful world of never being satisfied with the game the development company released!

TLDR: There are no specific rules, come up with good pricing, placement and reasons, and be prepared to defend those reasons and sometimes admit they might be too far fetched and have at it!

So your questions are something that are not very easy to answer. As you can see from #945 (and there are probably others) there is not a consensus on how things should be. However, my opinion (and others may disagree), is that mostly everything after Apollo can be adjusted based on gameplay and alternate possibilites.

We still want things to have realism as the core focus. So while it "might" have been possible to have a Mars mission in 1975, it is incredibly unlikely and something in the 80's makes a lot more sense. If the US decided not to pursue the Shuttle, things would have been very different. A lot of what has been looked at is based on the FANTASTIC alternate history Eyes Turned Skyward: https://www.alternatehistory.com/wiki/doku.php?id=timelines:eyes_turned_skyward_spacecraft_and_launch_vehicle_technical_data

This is a very plausible way of how things could have happened in an alternate reality. These are the types of decisions and setups that we not only encourage, but love. If you can come up with the reasons why you have chosen to put specific parts in nodes and come up with sensible costs based on what has come before, what could have happened, and what came afterwards, it would be fantastic.

RP-1 is a great mod, right up until you land people on the Moon. Then you can build a large space station, but it is rather linear in how things were done historically. Some stuff like Real ISRU attempt to bridge some of that gap, but there are not a lot of options. If you could develop some alternate reasons for parts to be put into an interesting place during the time frame of like 1975 to 2000, it would be phenomenal.

**Comment by karlack26** [comment]
Haha not being satisfied with a game that was released, started about 30 hours into Oblivion back in 2006.
Thanks for the reply that's what i roughly thought  the general conses was for parts post Apollo.  But wanted a more concrete conformation. Since i was going off of what RP-0 devs had posted on reddit and the KSP forums over the years. 
May i suggest you add your reply to the wiki under the section about configuring parts for RP-1  to further help any would be part configuring enthusiast. 
Another question
Is pricing still 1 credit = 1000 1965 USD
i dug that up in the KSP forums. I knew i had read at some point, how things were priced. 
But could not recall the exact amount.

**Comment by pap1723** [comment]
Good idea! And yes,that is the conversion
